src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/HistoryOfAccessionsEntity.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.commons.accession.persistence;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import java.sql.Date;
+
+@Entity
+public class HistoryOfAccessionsEntity {
```

Comment:
`AccessionHistoryEntry` instead of `HistoryOfAccessions` would be more descriptive because an accession could go through multiple statuses. Also note the singular instead of plural, because each entry applies to a single accession. Also applies to the DTO.